.github/workflows/ci.yml

```diff
@@ -593,10 +593,53 @@ jobs:
         DD_API_KEY_3: ${{ secrets.DD_API_KEY_3 }}
         DD_APP_KEY_3: ${{ secrets.DD_APP_KEY_3 }}
 
+    - name: Compress logs
```

Comment:
IIUC, what we really test here is the ability of the agent to handle open telemetry trace, right ?

Reply:
They are testing different "Open Telemetry scenarios".
There is a good documentation:

- https://datadoghq.atlassian.net/wiki/spaces/OTEL/pages/3012035497/OpenTelemetry+E2E+System+Tests
- https://docs.google.com/document/d/1uy73XwAc6QrcmfOFYLCTpOSgYZnGKOXlP9cnrWTIllo/edit#heading=h.6beuftc20yw3

When I'm done with the Open Telemetry and DB Integrations PRs, I'm thinking of documenting what I'm doing and taking the existing documentation about system-tests and Open Telemetry and bringing it into our confluence/system-tests